backend/src/main/java/gov/cdc/usds/simplereport/service/UploadService.java

```diff
@@ -62,6 +62,17 @@ public UploadService(PersonService ps) {
         }
     }
 
+    public String getRow(Map<String, String> row, String name, boolean isRequired) {
+        String value = row.get(name);
+        if (!isRequired) {
+            return value;
+        }
+        if(value == null || value.trim().isEmpty()) {
+            throw new IllegalGraphqlArgumentException(name + " is required.");
```

Comment:
+1 that we now include which column is invalid! Would be nice if we could include the row number somehow.